README.md

```diff
@@ -49,39 +53,82 @@ module "github_token_minter" {
+The two that are not optional are the `GITHUB_APP_ID` and the `GITHUB_PRIVATE_KEY`.
 
-All three of those values come from [creating and installing](https://docs.github.com/en/apps/creating-github-apps/creating-github-apps/creating-a-github-app) your GitHub app within your organization. These values are sensitive and must be treated as secrets. The default Terraform module stores them in Secret Manager and they are mapped into the Cloud Run service from there.
+Both of those values come from [creating and installing](https://docs.github.com/en/apps/creating-github-apps/creating-github-apps/creating-a-github-app) your GitHub app within your organization. These values are sensitive and must be treated as secrets. The default Terraform module stores them in Secret Manager and they are mapped into the Cloud Run service from there.
 
 ENV VAR name                       | Description
 -----------------------------------| -----------
 GITHUB_APP_ID                      | GitHub App Id created following the app creation instructions at <https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app>
 GITHUB_PRIVATE_KEY                 | Private key generated as part of the GitHub App creation
-GITHUB_INSTALL_ID                  | GitHub Installation Id generated by installing the app via <https://docs.github.com/en/developers/apps/managing-github-apps/installing-github-apps>
 GITHUB_API_BASE_URL                | GitHubAPIBaseURL is the base URL for the GitHub installation. It should include the protocol (https://) and no trailing slashes.
 PORT                               | Port to run the service on. Defaults to `8080`
-CONFIGS_DIR                        | Location of repository configuration files. Defaults to `configs`
+CONFIGS_DIR                        | Location of local configuration files on the filesystem. Defaults to `configs`
 ISSUER_ALLOWLIST                   | The list of OIDC token issuers that GitHub Token Minter will accept. Defaults to accepting GitHub and Google tokens.
 JWKS_CACHE_DURATION                | The duration for which to cache the JWKS for an OIDC token issuer.
+REPO_CONFIG_PATH                   | The location within a repository to look for configuration files. Defaults to .github/minty.yaml
+ORG_CONFIG_REPO                    | The respository that contains the configuration file for an organization. Defaults to .google-github
+ORG_CONFIG_PATH                    | The location within an organization to look for configuration files. Defaults to minty.yaml
+REF                                | The ref (sha, branch, etc.) to look for configuration files at. Defaults to main
 
 ### Configuring Repository Access
 
+**NOTE**: We recommend that configuration for single repository access is stored within that repository and configuration that spans multiple repositories is stored in a protected organization repository, typically named `.google-github`.
```

Comment:
Should we call this `.minty` instead?

Reply:
No. We went back and forth on this and landed on `.google-github`. I'll be putting other stuff in there long term that won't be minty related.

Reply:
Right, that seems to be specific to Google. For other people who aren't Google, it might be better to default to .minty and we alter to our own convention?

Reply:
Added #183